app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java

```diff
@@ -321,6 +321,7 @@ private void parseSusiResponse(SusiResponse susiResponse) {
             isWebSearch = susiResponse.getAnswers().get(0).getActions().get(1).getType().equals("websearch");
             datumList = susiResponse.getAnswers().get(0).getData();
             webSearch = susiResponse.getAnswers().get(0).getActions().get(1).getQuery();
+            answer = answer+"#"+webSearch;
```

Comment:
Currently when we retrieve old messages then variable webSearch contains only one only which is last retrieved websearch. So [webquery](https://github.com/fossasia/susi_android/blob/development/app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/ChatFeedRecyclerAdapter.java#L486)  can have only that single value. Thats why retrieved messages show same webresult. Here I attached each websearch with answer so that when recyclerview inflate with text answer [here](https://github.com/fossasia/susi_android/blob/development/app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/ChatFeedRecyclerAdapter.java#L483), it will show corresponding websearch.